src/main/java/com/alkemy/ong/infrastructure/rest/request/CategoryPostRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.alkemy.ong.infrastructure.rest.request;
+
+import com.alkemy.ong.infrastructure.rest.request.validation.CharactersWithWhitespaces;
+import javax.validation.constraints.NotEmpty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class CategoryPostRequest {
+
+  @NotEmpty
+  @CharactersWithWhitespaces(message = "Category name must contain only spaces and letters.")
```

Comment:
MINOR: para ser consistente con los otros mensajes dejaría: "Name must contain only spaces and letters."